diagrams/designArchitecture.puml

```diff
@@ -0,0 +1,41 @@
+@startuml
+
+class Command
+class Show
+class Storage
+class WatchTime
+class Duke
+class Ui
+interface SaveState
+class ErrorHandling
+class ShowList
+class InputParser
+class StringOperations
+
+Duke *--> "1" Ui
+Duke *--> "1" ShowList
+Duke *--> "1" Storage
+Duke *--> "1" WatchTime
+Duke --> "1" InputParser
+
+Storage ..|> SaveState
+Storage --> "1" WatchTime
+Show <..> Ui
+Ui ..> ShowList
+Ui ..> ErrorHandling
+InputParser --> StringOperations
+InputParser --> "0..1" Ui
+ShowList "1" *--> "*" Show
+InputParser --> Command
+
+package "Command" {
+Command <|-- AddCommand
+Command <|-- EditCommand
+Command <|-- DeleteCommand
+Command <|-- OtherCommands
+class AddCommand
+class EditCommand
+class DeleteCommand
+class OtherCommands
+}
+@enduml
```

Comment:
<img width="650" alt="Capture" src="https://user-images.githubusercontent.com/60425261/97666823-6c1c5480-1ab9-11eb-8061-76fb13158f32.PNG">
Hi guys, remember to turn off the icons feature to give a more standardized UML diagram. I think prof gave some suggestions on how to change it in the forum.